cmd/infracost/testdata/output_format_azure_repos_comment/output_format_azure_repos_comment.golden

```diff
@@ -219,7 +219,7 @@ Percent: +100%
 ──────────────────────────────────
 
 The following projects have no cost estimate changes: infracost/infracost/cmd/infracost/testdata/terraform_v0.14_nochange_plan.json
-Run infracost breakdown to see their breakdown.
+Run infracost breakdown --path /path/to/code to see their breakdown.
```

Comment:
I would assume users would try to copy the command, but it might not be obvious what to copy. If we put it in the end that might be clearer. What do you think?
```suggestion
Run the following command to see their breakdown: infracost breakdown --path=/path/to/code
```